backend/src/lang/lang.controller.ts

```diff
@@ -3,12 +3,14 @@ import { DialogFlowService } from './dialog-flow/dialog-flow.service';
 import { AudioIntentParams, TextIntentParams, TextIntentBody } from './lang.dto';
 import { DatabaseLangService } from '../connectors/database-lang.service';
 import { IntentStrategy } from './intents/strategy/strategy.intent';
+import IntentConfig from './intents/IntentConfig';
+import { DIALOGFLOW_INTENT_IDS } from './dialog-flow/dialogflow-structure';
 
 /**
  * The settings of recorded android files
  * @type {any}
  */
-const ANDROID_AUDIO_SETTINGS: any = {
```

Comment:
Das any wegzulöschen macht zwar Sinn, sollte dann aber nicht auch der Type oben im JsDoc angepasst werden?

Reply:
Gute Frage, ich weiß nicht ob das so sinnvoll ist. Dadurch, dass ich das :any wegenommen habe, kann TypeScript den Type genau übernehmen. Der Type ist ja aber irgendwie sowas: { smaplingRate: string, ...} Gibt es Sinn das alles in die JSDocs zu packen?

Reply:
Hast du Recht. Dann lass es so.